inference/vllm_predictor.py

```diff
@@ -0,0 +1,57 @@
+from typing import AsyncGenerator, List, Union
+from predictor import Predictor
+from inference_config import InferenceConfig
+from transformers import TextIteratorStreamer
+from vllm.engine.arg_utils import AsyncEngineArgs
+from vllm.engine.async_llm_engine import AsyncLLMEngine
+from vllm.sampling_params import SamplingParams
+from vllm.utils import random_uuid
```

Comment:
Could we add this method to inference/util.py? I will also change the relevant methods in api_openai_backend to be called from here.
https://github.com/intel/llm-on-ray/blob/87ffd4f92336e49fcb54882ff331b9cb6aa6ef79/inference/api_openai_backend/router_app.py#L272

https://github.com/intel/llm-on-ray/blob/87ffd4f92336e49fcb54882ff331b9cb6aa6ef79/inference/api_openai_backend/router_app.py#L329

Reply:
I am not sure if random_uuid vllm used here is the same with the uuid you used. We can follow up later with another PR.